mm2src/database/stats_swaps.rs

```diff
@@ -13,7 +15,9 @@ const CREATE_STATS_SWAPS_TABLE: &str = "CREATE TABLE IF NOT EXISTS stats_swaps (
     finished_at INTEGER NOT NULL,
     maker_amount DECIMAL NOT NULL,
     taker_amount DECIMAL NOT NULL,
-    is_success INTEGER NOT NULL
+    is_success INTEGER NOT NULL,
+    maker_coin_usd_price DECIMAL,
+    taker_coin_usd_price DECIMAL
```

Comment:
One very important note: we need to perform SQL migration from the previous state of the `stats_swaps` table by adding these new `maker_coin_usd_price`, `taker_coin_usd_price` fields. It's needed because our clients might create the `stats_swaps` table without `maker_coin_usd_price`, `taker_coin_usd_price` and they even may have records in the table already.
Please check where the `CREATE_STATS_SWAPS_TABLE` query is executed, what stages of migration we have and add one more stage.
A few more hints:
* I guess the new migration will be seventh
* You need to add an SQL query that will change existing `stats_swaps` table by `ALTER TABLE` command

Please do the same for the `my_swaps` table as well 🙂